### PR TITLE
docs(storage): add examples to retrieve hash of blob

### PR DESCRIPTION
**Pull request by HemangChothani**
Issue [54]

### CODE REVIEW

**Review by mf2199 — CHANGES_REQUESTED** [review]
I believe the issue is about clarification, not just a code sample. The sample is fine as a supplement to a concise but clear description explaining in simple words the nature and handling of `crc32c` and `md5_hash` properties. Please add such description, and don't worry about perfecting the grammar and style - we can do that together. Be generous with words - they can be compactified later as well.

**Comment by HemangChothani** [comment]
@mf2199 @IlyaFaer  Is this description adequate?
`This returns the blob's CRC32C checksum. To retrieve the value, first 
    use a ``reload`` method of the Blob class which loads the blob's properties  
    from the server.`

File: google/cloud/storage/blob.py [hunk 1]
```diff
@@ -1881,6 +1881,22 @@ def update_storage_class(self, new_class, client=None):
...
+
+    Example:
+            Retrieve the crc32c hash of blob.
+
+            >>> from google.cloud import storage
+            >>> client = storage.Client()
+            >>> bucket = client.get_bucket("my-bucket-name")
+            >>> blob = bucket.blob('my-blob')
+
+            >>> blob.crc32c  #return None
```
Comment (IlyaFaer) [inline google/cloud/storage/blob.py]: There should be a space after `#` (standard and so on)

**Review by IlyaFaer — APPROVED** [review]
One nit. I think, it makes sense to ask googlers, if they need a test for this case. I'm not sure, but maybe, maybe, as it's documented

File: google/cloud/storage/blob.py (hunk 1 above)
Comment (HemangChothani) [reply google/cloud/storage/blob.py]: Any suggestion on description which i have mentioned above in comment

Comment (IlyaFaer) [reply google/cloud/storage/blob.py]: In fact, I don't think, it's needed. Example code speaks for itself. So, if I would have been doing this, I haven't comment it so detailed.

**Comment by HemangChothani** [comment]
@mf2199  please comment , if description is needed, than the following is  adequate or  any correction.

> @mf2199 @IlyaFaer Is this description adequate?
> ` This returns the crc32c checksum of blob. To retrieve the value, first use a ``reload`` method of the Blob class which loads the blob's properties from the server.`

**Comment by mf2199** [comment]
> @mf2199 @IlyaFaer Is this description adequate?
> `` This returns the blob's CRC32C checksum. To retrieve the value, first use a ``reload`` method of the Blob class which loads the blob's properties from the server. ``

@HemangChothani Edited your description, check it out.

**Comment by HemangChothani** [comment]
Merged.

**Pull request closed without merging**